scripts/fetch_manifests.sh

```diff
@@ -1,5 +1,6 @@
 #!/bin/bash
 
+set -x
```

Comment:
If you also want to log the manifests you might want to replace the ` > /tmp/manifests` below with `tee` ?

Reply:
Good idea! Added it